foia_hub/static/js/main.js

```diff
@@ -41,9 +42,17 @@ $(document).ready(function(){
         }
     }
     $bannerCloseButton.click(function(){
-        $("#notice").slideUp();
+        $("#notice").slideUp(function () {
+          times_toggled++
+            ga('send', {
```

Comment:
Yep, they're already added, but it's on this [line](https://github.com/18F/foia-hub/pull/737/files#diff-62cea810ef134ce73a2a312261b688b9R27)